superset-frontend/src/SqlLab/components/ResultSet/ResultSet.test.jsx

```diff
@@ -177,7 +179,7 @@ test('should render empty results', () => {
 test('should render cached query', () => {
   const wrapper = shallow(<ResultSet {...cachedQueryProps} />);
   const cachedData = [{ col1: 'a', col2: 'b' }];
-  wrapper.setState({ data: cachedData });
+  wrapper.setState({ data: cachedData }); // How to set state to Functional Component?
```

Comment:
@lyndsiWilliams How to set state to Functional Component?

Reply:
I believe this is setting the `query.results.data` as seen in `newProps` here:
https://github.com/apache/superset/blob/f62aa7b1b8cddd6f2655dbb657b730b5e27778df/superset-frontend/src/SqlLab/components/ResultSet/ResultSet.test.jsx#L88-L96
so you'll need to change the render's props in RTL to reflect that.